solver/src/settlement.rs

```diff
@@ -46,6 +46,20 @@ impl Trade {
         }
     }
 
+    // Returns the relative fee amount given the executed amount
+    // cf. https://github.com/gnosis/gp-v2-contracts/blob/964f1eb76f366f652db7f4c2cb5ff9bfa26eb2cd/src/contracts/GPv2Settlement.sol#L370-L371
+    pub fn fee(&self) -> U256 {
```

Comment:
This is the executed fee amount, right? Maybe easier to understand if is called that instead of relative fee.